### ISSUE TITLE
Final Presentation - Detailed Design

### ISSUE THREAD

**Issue by nnepa066** [nnepa066/Tracker-Mapper#103]
- H/W & S/W Designs: Details of H/W and S/W design. Highlight any changes made to the existing design, etc. 
- Design choices: Identified alternatives and advantages of using your design. Why is your solution better than alternatives present in the market? Compare performance.
- Project involving ML/AI parts demonstrate your accuracy scores, training loss, validation loss, various graphs, etc. Highlight the model used.
- Provide what modifications did you do to the existing model if any.
- State hazard and legal verification
